src/sentry/integrations/slack/message_builder/base/base.py

```diff
@@ -93,9 +94,10 @@ def _build(
 
         markdown_in = ["text"]
         if self.escape_text:
-            text = escape_slack_text(
-                escape_slack_text(text)
-            )  # Slack will un-escape so we have to double escape
+            text = escape_slack_text(text)
+            # for unfurling messages we need to double escape the text
```

Comment:
Why?

Reply:
Ah you said you don't know. I might update this comment with something like 

```
XXX(scefali): Not sure why we actually need to do this. 
If we figure out why this is required we should note it
here because it's quite strange
```

Reply:
@evanpurkhiser missed this, will update the comment